php/Terminus/User.php

```diff
@@ -73,12 +75,21 @@ public function sites($organization=null) {
     return $response['data'];
   }
 
-  public function getId() {
-    return $this->id;
+  public function instruments() {
+    $instruments       = new Instruments(array('user' => $this));
```

Comment:
Alright, but how can the original configuration give an error? $instruments is private. You can't access it uninstantiated, and it's pretty useless without fetch().